cmd/api/src/database/migration/migrations/v5.0.11.sql

```diff
@@ -0,0 +1,26 @@
+-- Copyright 2023 Specter Ops, Inc.
+--
+-- Licensed under the Apache License, Version 2.0
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+DELETE
+FROM risk_posture_stats
+WHERE created_at > TIMESTAMP WITH TIME ZONE '2023-08-30 19:00:00+00'
+  AND created_at < TIMESTAMP WITH TIME ZONE '2023-08-31 21:30:00+00';
+
+
+DELETE
+FROM risk_counts
+WHERE created_at > TIMESTAMP WITH TIME ZONE '2023-08-30 19:00:00+00'
+  AND created_at < TIMESTAMP WITH TIME ZONE '2023-08-31 21:30:00+00';
```

Comment:
Neither of these exist in BHCE, which makes these migrations likely to break when upgrading a live version of BHCE. We will only want to run the DELETE statements _if_ the tables exist. I'm not entirely sure the best way to do this, so we can discuss outside of the PR to find the best solution here.